examples/advanced/flutter/style_dictionary/lib/style_dictionary_asset.dart

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Generally, the files built with style dictionary like these are not committed. Should we add the lib/ directory to the gitignore of this example? Or do these files need to get committed?